source/edge-server/configure.txt

```diff
@@ -18,113 +18,95 @@ Configure Edge Server
    :depth: 2
    :class: singlecol
 
-Edge Server is a "local" server that sits between your client devices and 
-MongoDB Atlas App Services. For most use cases, you'll provision hardware 
-on-premises to host the Edge Server. While you're developing your Edge 
-Server application, you may host it on a cloud server for convenience.
+After you create an Edge Server instance in Atlas, you must configure the 
+hardware that runs the Edge Server instance. For most use cases, you'll 
+provision hardware on-premises to host the Edge Server instance. While you're 
+developing your Edge Server application, you may host it on a cloud server 
+for convenience.
 
-Client devices on-prem connect to Edge Server. Your Edge Server connects 
-to App Services. Clients do not connect to App Services directly.
+Client devices on-prem connect to the Edge Server instance. Your instance 
+connects to the Edge Server Service or Device Sync Service, depending on
```

Comment:
One question, so I think technically, you can start up device sync + edge server on a triggers app/service. Are we trying to refer to any service with Device Sync enabled as a "Device Sync Service"? And not specifically services created via the Device Sync service via MMS?

Reply:
I have language introducing the two "types" of services on the "Manage Edge Servers in Atlas" page, which you can see here on the feature branch: [Manage Edge Servers in Atlas](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/edge-server-public-preview-docs/edge-server/manage-edge-servers/)

I didn't really get into the types of App Services that can have Device Sync. I _think_ that's outside the scope of the Edge Server docs, but if you think it's important to get into these details, I have a placeholder to populate the "[Create an App Service](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/edge-server-public-preview-docs/edge-server/manage-edge-servers/#create-a-service-for-edge-server)" procedure once I can test with the UI. I could add that detail there.

IMO, it's a bit messy with all the different entrypoints, so I was trying to hand-wave away the product types a little. By the time a customer get to this page, they should have already set up the relevant service based on the "Manage" page. I just left this intro at the top in case someone skips to the "Configure" page and thinks they can start here.